### PR TITLE
chore: adds issue template for bugs and feature requests

### PR DESCRIPTION
**Pull request by timbru31**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+**Desktop (please complete the following information):**
+
+- OS: <!-- [e.g. iOS] -->
+- Browser <!-- [e.g. chrome, safari] -->
+- Version <!-- [e.g. 22] -->
+
+**Smartphone (please complete the following information):**
+
+- Device: <!-- [e.g. iPhone6]
```
Comment (luismeyer) [inline .github/ISSUE_TEMPLATE/bug_report.md:37]: Missing comment ending

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+name: 🚀 - Feature request
+about: Suggest an idea for this project
+title: ""
+labels: enhancement
+assignees: ""
+---
+
+**Is your feature request related to a problem? Please describe.**
+
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] --><!--
```
Comment (luismeyer) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: Wrong comment opening. At the end of the line

**Review by luismeyer — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (timbru31) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Fixed

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (timbru31) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Weird, VSCode did not complain. Fixed.

**Review by luismeyer — APPROVED** [review]

**Pull request merged**